### PR TITLE
Start a "How We Work" document

### PR DESCRIPTION
**Pull request by waldoj**
A great deal more should go into a document like this, but I'd like to start with something that should be documented now, so that we can all be on the same page about norms and expectations for pull requests.

### CODE REVIEW

**Comment by waldoj** [comment]
@jusdino, I'm hoping that you can take a look at this and see if it aligns with how you'd like to be working. And to the extent to which it doesn't, please propose some edits! And then once you and I agree there, @isabeleliassen, you can see what you think of it.

And this could reasonably change within a few sprints, after @jlkravitz gets settled.

**Comment by jusdino** [comment]
Oh, thank you! We were just discussing items that intersect with this, today. We did decide, with @jlkravitz, to transition to per-feature PR's for now. We also have a story in progress right now to document API design guidelines as well, which will be part of a similar markdown document, as we build up more working conventions.

File: HOW_WE_WORK.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# How We Work
+
+## Vendor pull requests
+
+InspiringApps will work on their own fork of the repository. They will file a single pull request at the end of each sprint that includes all work completed in that sprint, as presented at the sprint review. It should be filed within one working day of the end of the sprint, e.g. if the sprint ends on a Thursday, it should be filed by the end of the day on Friday. That pull request will be titled for the sprint (e.g., `Sprint 15`), and the description will summarize the work of the sprint. That will lead with the sprint goal, and then list each completed user story. Each completed user story will include the title, will link back to the story in the backlog, and should include an illustrative screenshot (where applicable).
```
Comment (jusdino) [inline HOW_WE_WORK.md:5]: We did agree to some different practices, as of this morning:

- We will open pull requests for each feature, as they are completed, after we have completed our own QA/review process
- Pull request names should reflect the feature they encapsulate
- We will open pull requests from a feature branch in the IA fork to the `development` branch in the CSG repo
- CSG will periodically merge their `development` branch into `main` to promote updates in their staging (UAT) environment to production

**Review by jusdino — CHANGES_REQUESTED** [review]

**Comment by waldoj** [comment]
Great! I've updated the document to reflect that agreement. How does this look, @jusdino?

**Comment by jusdino** [comment]
Looks great!

**Review by jusdino — APPROVED** [review]

**Review by isabeleliassen — APPROVED** [review]
Looks good to me! We should have a more thorough version of a How We Work document for the team to look at in about a week.

**Pull request merged**